client/cypress/e2e/_13metrics.cy.ts

```diff
@@ -0,0 +1,192 @@
+          .invoke('text')
+          .should('eq', isMobile || isTablet ? 'E3 Allocation' : 'Epoch 3 Allocation Window');
+      });
+
+      it('"General" section title is visible and has correct text', () => {
+        cy.get('[data-test=MetricsGeneral__header__title]').should('be.visible');
+        cy.get('[data-test=MetricsGeneral__header__title]')
+          .invoke('text')
+          .should('eq', 'General metrics');
+      });
+
+      it('"Epoch" section title has "OPEN" badge when AW is open', () => {
+        cy.get('[data-test=MetricsEpochHeader__openBadge]').should('be.visible');
+        cy.get('[data-test=MetricsEpochHeader__openBadge]').invoke('text').should('eq', 'Open');
+        cy.get('[data-test=MetricsEpochHeader__openBadge]')
+          .then($el => $el.css('backgroundColor'))
+          .should('be.colored', '#2d9b87');
+        cy.get('[data-test=MetricsEpochHeader__openBadge]')
+          .then($el => $el.css('textTransform'))
+          .should('eq', 'uppercase');
+      });
+
+      it('User is able to change epoch via arrow buttons', () => {
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows]').should('be.visible');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrow]').click();
+        cy.get('[data-test=MetricsEpochHeader__title]')
+          .invoke('text')
+          .should('eq', isMobile || isTablet ? 'E2 Allocation' : 'Epoch 2 Allocation Window');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__rightArrow]').click();
+        cy.get('[data-test=MetricsEpochHeader__title]')
+          .invoke('text')
+          .should('eq', isMobile || isTablet ? 'E3 Allocation' : 'Epoch 3 Allocation Window');
+      });
+
+      it('"Epoch" section title has epoch dates range when epoch is past', () => {
+        cy.get('[data-test=MetricsEpochHeader__openBadge]').should('be.visible');
+        cy.get('[data-test=MetricsEpochHeader__epochDuration]').should('not.exist');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__leftArrow]').click();
+        cy.get('[data-test=MetricsEpochHeader__openBadge]').should('not.exist');
+        cy.get('[data-test=MetricsEpochHeader__epochDuration]').should('be.visible');
+      });
+
+      it('Next arrow button is disabled when there is no more next epochs', () => {
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__rightArrow]')
+          .then($el => $el.css('backgroundColor'))
+          .should('be.colored', '#fafafa');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__rightArrowSvg] path')
+          .then($el => $el.css('fill'))
+          .should('be.colored', '#ebebeb');
+        cy.get('[data-test=MetricsEpochHeader__title]')
+          .invoke('text')
+          .should('eq', isMobile || isTablet ? 'E3 Allocation' : 'Epoch 3 Allocation Window');
+        cy.get('[data-test=MetricsEpochHeader__NavigationArrows__rightArrow]').click({
+          force: true,
```

Comment:
Why? `force` means it's not actually clickable for the user, indicating a problem with UI.

Reply:
Fixed